web/client/plugins/QueryPanel.jsx

```diff
@@ -435,6 +446,43 @@ class QueryPanel extends React.Component {
  *        },
  *        "customItemClassName": "customItemClassName"
  *    }
+ *
+ * @example
+ * // customize the QueryPanels UI via plugin(s)
+ * import {createPlugin} from "../utils/PluginsUtils";
+ *
+ * export default createPlugin('QueryPanelCustomizations', {
+ *     component: () => null,
+ *     containers: {
+ *         QueryPanel: [
+ *             {
```

Comment:
correct me if I'm wrong
```suggestion
 *             // Hide the attribute filter by injecting a `component: () => null` for one of the default targets, e.g. `attributeFilter`.
 *             {
```